### PR TITLE
json generation fully implemented

### PR DESCRIPTION
**Pull request by kyle-koch**
press the button Kronk!

### CODE REVIEW

File: server/src/Routes/Json.ts
Comment (jmidd2) [inline server/src/Routes/Json.ts:1]: You left some unneeded words in the console.

**Comment by jmidd2** [comment]
The functionality is all good. The location and name of files needs to be moved around though. The exportDbToJson that you created is a service it's a part of our business logic. Business logic and services live in the Services folder.

**Review by jmidd2 — CHANGES_REQUESTED** [review]
The functionality is all good. The location and name of files needs to be moved around though. The exportDbToJson that you created is a service it's a part of our business logic. Business logic and services live in the Services folder. the Json.ts routes file has some unneeded words in it as well.

**Comment by kyle-koch** [comment]
Hopefully this is better 😅

**Review by jmidd2 — APPROVED** [review]

**Pull request merged**